Config/Inc/config.h

```diff
@@ -12,10 +12,13 @@ typedef enum {OFF = 0, ON} State;
 
 // Wheel diameter in meters
 #define WHEEL_DIAMETER 0.4f
+
 // Any sort of gear reduction from the motor to the wheels
 // Defined as motor velocity / wheel velocity
 #define GEAR_REDUCTION 1.0f
 
+#define PRECHARGE_ARRAY_DELAY(err) (OSTimeDlyHMSM(0, 0, 5, 0, OS_OPT_TIME_HMSM_STRICT, &err))
```

Comment:
Oh I think I said it wrong. I mainly just meant make the number 5 into the macro. For the number of seconds we need to delay. That's the only thing we need to change really